sycl/include/CL/sycl/detail/pi.h

```diff
@@ -321,23 +405,40 @@ pi_result piQueueFinish(pi_queue command_queue);
 //
 // Memory
 //
-pi_mem piMemCreate( // TODO: change interface to return error code
+pi_mem piMemBufferCreate( // TODO: change interface to return error code
   pi_context   context,
   pi_mem_flags flags,
   size_t       size,
   void *       host_ptr,
   pi_result *  errcode_ret);
 
+pi_mem piMemImageCreate( // TODO: change interface to return error code
+  pi_context              context,
+  pi_mem_flags            flags,
+  const pi_image_format * image_format,
+  const pi_image_desc *   image_desc,
+  void *                  host_ptr,
+  pi_result *             errcode_ret);
+
 pi_result piMemGetInfo(
```

Comment:
Given you are adding piMemImageGetInfo below, I assume this interface is not OK for images, and then it should be changed to piMemBufferGetInfo

Reply:
piMemImageGetInfo works for both images and buffers and provides common info. piMemImageGetInfo provides image specific info.

Reply:
OK